src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -180,13 +168,10 @@ public static void IsZipSameAsDir(Stream archiveFile, String directory, ZipArchi
 
                         if (!dontCheckTimes)
                         {
-                            Double offBy = (file.LastModifiedDate - entry.LastWriteTime.DateTime).TotalSeconds;
-                            Assert.True(
-                                (offBy >= -2 && offBy <= 2) ||
-                                // Temporary adjustment for active issue 1326
-                                ((offBy >= 3598 && offBy <= 3602)),
-                                String.Format("{0}, {1}, {2}", file.LastModifiedDate.ToString(), entry.LastWriteTime.DateTime.ToString(), file.FullName));
-                        }
+                            DateTime lower = file.LastModifiedDate.AddSeconds(-2);
+                            DateTime upper = file.LastModifiedDate.AddSeconds(2);
```

Comment:
I know you're just cleaning up existing code, but while you're at it, would it be worth putting `2` in a `const` and/or adding a comment explaining that the zip timestamp resolution is 2 seconds, hence the reason for the range assert?